doc/user_guide/user_guide.rst

```diff
@@ -0,0 +1,165 @@
+.. _user_guide:
+
+:octicon:`person` User Guide
+============================
+
+This project contains a Python library for describing Exasol error messages.
+This library lets you define errors with a uniform set of attributes.
+Furthermore, the error message is implemented to be parseable,
+so that you can extract an error catalog from the code.
```

Comment:
```suggestion
This project contains a Python library for describing error messages in Exasol Python projects.
This library lets you define errors with a uniform set of attributes.
Furthermore, the error message is implemented to be parseable,
so that you can extract an error catalog from the code.
```

Reply:
I am not 100% happy with _describing_.
* How about _defining_ 
* or _defining, creating, using, and displaying_, though I'm aware, that the actual _display_ is probably up to the application rather than the ERP project.

Maybe the attribute _unique_ is also important here and should be mentioned?